### ISSUE TITLE
Bridged network of nanos

### ISSUE THREAD

**Issue by QiZhang1997** [nanovms/nanos#1363]
Recently I followed this tutorial [here](https://nanovms.gitbook.io/ops/networking#bridged-network) to build bridged network of nanos. I noticed that I should have an Ethernet connection if I want to setup this networking mode. So I connected a router on the campus network and connected the computer to the router with the network cable. I'm sure my computer is connected to Ethernet. But I still got the following error after running `sudo ops net setup` command:
```
no DHCP packet received within 10s
```
I don't know where the problem is. Does anyone know what to do to solve this problem?

**Comment by eyberg** [comment]
I think that tutorial/code might need to be updated - we are tracking this over here -> https://github.com/nanovms/ops/issues/836

**Comment by QiZhang1997** [comment]
@eyberg Thanks! I followed the solution here https://github.com/nanovms/ops/issues/188. Assign the next valid IP to bridge instead of using DHCP. After that, I found that the host’s network was broken, but the unikernel virtual machine had a network connection. I want to use the terminal on the host and unikernel to communicate over the network, or two unikernel virtual machines to communicate over the network. I want to ask whether nanos supports starting two virtual machines in bridge mode at the same time.
